code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm

```diff
@@ -93,29 +93,6 @@
 	if(mind)
 		mind.name = name
 
-// ***************************************
-// *********** Icon
-// ***************************************
-/mob/living/carbon/xenomorph/queen/Topic(href, href_list)
```

Comment:
same here, doenst this make queen lose overwatch ?

Reply:
Its inherited from /xenomorph @SplinterGP